src/components/ActiveAccount.js

```diff
@@ -1,38 +1,45 @@
 import React from 'react';
 import lang from 'languages';
 import { Flex, Text, Box } from '@makerdao/ui-components-core';
-import Jazzicon, { jsNumberForAddress } from 'react-jazzicon';
 
 import { ReactComponent as CaratDownIcon } from 'images/carat-down.svg';
 
 import { getWebClientProviderName } from 'utils/web3';
 import { cutMiddle } from 'utils/ui';
 
 const ActiveAccount = ({
-  currentAccount,
+  address,
+  type,
   textColor = 'darkPurple',
   iconSize = 22,
   t = 'body',
   addressTextStyle = 'body',
   readOnly,
   ...rest
 }) => {
-  const { address, type } = currentAccount;
-  if (!currentAccount) return null;
   const providerType =
     type === 'browser' ? getWebClientProviderName(type) : type;
   return (
-    <Flex alignItems="center" {...rest}>
-      <Jazzicon diameter={iconSize} seed={jsNumberForAddress(address)} />
+    <Flex justifyContent="space-between" alignItems="center" {...rest}>
+      <Text
+        pr="xs"
+        fontSize="0.8rem"
+        color={address ? 'makerTeal' : 'makerOrange'}
+      >
+        &#11044;
+      </Text>
+
       <Box ml="xs" mr="auto">
         <Text t={t} color={textColor}>
-          {lang.providers[providerType]}
+          {address ? lang.providers[providerType] : lang.sidebar.no_wallet}
         </Text>
       </Box>
       <Box ml="s">
-        <Text t={addressTextStyle} color={textColor}>
-          {cutMiddle(address, 7, 5)}
-        </Text>
+        {address ? (
+          <Text t={addressTextStyle} color={textColor}>
+            {cutMiddle(address, 7, 5)}
+          </Text>
+        ) : null}
       </Box>
```

Comment:
These are the updates to the sidebar wallet dropdown. Is it necessary to hide this component when on `/borrow` as we would, in that case, have two wallet logins?

Reply:
The Save pages are going to be revised by the design team so they make sense when integrated with the CDP portal. Probably we'll show this component again in an appropriate place. For now, just remove the placeholder account box that shows up in the upper right of the Save pages.